GOVERNANCE.md

```diff
@@ -15,6 +15,9 @@ Current core team members are:
 | Geraldine Van der Auwera | Broad Institute | [vdauwera](https://github.com/vdauwera) |
 | Christopher Llanwarne | Broad Institute | [cjllanwarne](https://github.com/cjllanwarne) |
 | John Didion | DNAnexus | [jdidion](https://github.com/jdidion) |
+| Michael Franklin | Australian Centre for Population Genomics | [illusional](https://github.com/illusional) |
+| Amy Paguirigan | Fred Hutch | [vortexing](https://github.com/vortexing) |
+| Ruben Vorderman | BioWDL | [rhpvorderman](https://github.com/rhpvorderman) |
```

Comment:
My official affiliation is "Leiden University Medical Center". The other people of BioWDL work there too.